index.js

```diff
@@ -36,9 +36,11 @@ commonWordsArray.forEach(function commonWordAdd(w) {
 
 module.exports = function _export(options) {
   var url = options.url
+  var returnSource = false
   if (options.body) {
-    return analyze(url, options.body)
+    return analyze(url, options.body, options.returnSource || returnSource)
```

Comment:
in the case of nytimes.com, we use `cfscrape` to get the content. so we want schenkerian to return the source when we pass in the body